### PR TITLE
Update zh-Hans.resx

### PR DESCRIPTION
**Pull request by yhpopcom**
添加了中文翻译
Added Chinese translation

### CODE REVIEW

File: PalCalc.UI/Localization/Localizations/zh-Hans.resx [hunk 1]
```diff
@@ -117,10 +117,628 @@
...
+  </data>
+  <data name="LC_PAL_LOC_CUSTOM" xml:space="preserve">
+    <value>(Custom)</value>
+  </data>
+  <data name="LC_PAL_LOC_PALBOX" xml:space="preserve">
+    <value>帕鲁终端</value>
+  </data>
+  <data name="LC_PAL_LOC_PARTY" xml:space="preserve">
+    <value>公会</value>
+  </data>
```
Comment (tylercamp) [inline PalCalc.UI/Localization/Localizations/zh-Hans.resx:402]: 队伍
小队

File: PalCalc.UI/Localization/Localizations/zh-Hans.resx [hunk 2]
```diff
@@ -117,10 +117,628 @@
...
+  <data name="LC_TRAITS_COUNT_EMPTY" xml:space="preserve">
+    <value>没有被动技能</value>
+  </data>
+  <data name="LC_TRAITS_COUNT_RANDOM" xml:space="preserve">
+    <value>{NumRandom} 随机</value>
+  </data>
+  <data name="LC_TRAITS_COUNT_UNRECOGNIZED" xml:space="preserve">
+    <value>{NumUnrecognized} 被忽略的</value>
+  </data>
```
Comment (tylercamp) [inline PalCalc.UI/Localization/Localizations/zh-Hans.resx:689]: {NumUnrecognized} 未识别的

File: PalCalc.UI/Localization/Localizations/zh-Hans.resx [hunk 3]
```diff
@@ -117,10 +117,628 @@
...
+  <data name="LC_RESULT_IV_ATTACK" xml:space="preserve">
+    <value>IV 攻击</value>
+  </data>
+  <data name="LC_RESULT_IV_AVERAGE" xml:space="preserve">
+    <value>IV AVG</value>
+  </data>
+  <data name="LC_RESULT_IV_DEFENSE" xml:space="preserve">
+    <value>IV DEF</value>
+  </data>
```
Comment (tylercamp) [inline PalCalc.UI/Localization/Localizations/zh-Hans.resx:460]: IV 防御

File: PalCalc.UI/Localization/Localizations/zh-Hans.resx [hunk 4]
```diff
@@ -117,10 +117,628 @@
...
+  <data name="LC_RESULT_INPUT_LOCS" xml:space="preserve">
+    <value>输入位置</value>
+  </data>
+  <data name="LC_RESULT_IV_ATTACK" xml:space="preserve">
+    <value>IV 攻击</value>
+  </data>
+  <data name="LC_RESULT_IV_AVERAGE" xml:space="preserve">
+    <value>IV AVG</value>
+  </data>
```
Comment (tylercamp) [inline PalCalc.UI/Localization/Localizations/zh-Hans.resx:457]: IV 平均

File: PalCalc.UI/Localization/Localizations/zh-Hans.resx [hunk 5]
```diff
@@ -117,10 +117,628 @@
...
+  <data name="LC_SAVE_GAME" xml:space="preserve">
+    <value>游戏</value>
+  </data>
+  <data name="LC_SAVE_GAME_INVALID" xml:space="preserve">
+    <value>选定的保存文件无效!</value>
+  </data>
+  <data name="LC_SAVE_GAME_LBL" xml:space="preserve">
+    <value>{PlayerName} 在第 {PlayerLevel} 天 {WorldName} 达到 {DayNumber}等级</value>
+  </data>
```
Comment (tylercamp) [inline PalCalc.UI/Localization/Localizations/zh-Hans.resx:541]: {PlayerName} 在 {WorldName} 第 {DayNumber} 天，等级 {PlayerLevel}

File: PalCalc.UI/Localization/Localizations/zh-Hans.resx [hunk 6]
```diff
@@ -117,10 +117,628 @@
...
+  <data name="LC_SAVE_GAME_LBL_NO_METADATA" xml:space="preserve">
+    <value>{GameId} (无法读取元数据)</value>
+  </data>
+  <data name="LC_SAVE_GAME_LBL_SERVER" xml:space="preserve">
+    <value>{WorldName} 天 {DayNumber}</value>
+  </data>
```
Comment (tylercamp) [inline PalCalc.UI/Localization/Localizations/zh-Hans.resx:547]: {WorldName} 第 {DayNumber} 天

File: PalCalc.UI/Localization/Localizations/zh-Hans.resx [hunk 7]
```diff
@@ -117,10 +117,628 @@
...
+  <data name="LC_RESULT_LIST_TITLE_EMPTY" xml:space="preserve">
+    <value>结果</value>
+  </data>
+  <data name="LC_RESULT_NUM_STEPS" xml:space="preserve">
+    <value># 步骤</value>
+  </data>
+  <data name="LC_RESULT_TIME_ESTIMATE" xml:space="preserve">
+    <value>Effort</value>
+  </data>
```
Comment (tylercamp) [inline PalCalc.UI/Localization/Localizations/zh-Hans.resx:478]: 耗时

**Review by tylercamp — COMMENTED** [review]
Thank you for the updates! Would you like your name to be included in the credits, in the "About" window? If so, let me know what name to use

I also checked the translations and have a few suggestions from ChatGPT, I left comments  on GitHub here

(It's your choice whether to review and update the text I checked, I can merge the current translations without any updates)

**Comment by yhpopcom** [comment]
> 感谢您的更新！您是否希望在“关于”窗口中的致谢部分中包含您的姓名？如果是，请告诉我要使用什么名字
> 
> 我还检查了翻译，并从 ChatGPT 那里得到了一些建议，我在 GitHub 上留下了评论
> 
> （您可以选择是否审阅和更新我检查过的文本，我可以合并当前的翻译而不做任何更新）

Thanks for merging the updates after checking
 感谢 可以合并检查后的更新
by 在这251秒赌上我的一切

**Comment by tylercamp** [comment]
(Since you haven't commented about putting your name in the credits, or about the suggestions I left, I'll merge your PR as-is later today without including your name in credits. Let me know if you'd like to change that later)

**Comment by yhpopcom** [comment]
Ok you may not have seen my comment
You can update the checked text
Name in the credits (在这251秒赌上我的一起)
🤦‍♂️

**Comment by tylercamp** [comment]
@yhpopcom Ok, I'll include that in the next update

**Comment by Narugakuruga** [comment]
I really appreciate the help. But apparently your translation is a mixture of machine translation and manually polishing. It made me and other people confused in some way. Can you fix that?

**Comment by tylercamp** [comment]
@Narugakuruga Sure, should I revert my changes and use the original text from @yhpopcom instead? Or are there other changes I should make?

**Comment by Narugakuruga** [comment]
I have better idea in a standalone Pull request. 
Used the software in English for a while I have understand the phrase and can interpret it naturally. 
It's not about the changes, the whole UI look alien I'm certain it's machine translation.

**Pull request merged**